docs/our_work/cvd_pathways.md

```diff
@@ -0,0 +1,126 @@
+---
```

Comment:
wont let me comment it directly on the image files for some reason but, seeing as your project has three images, would you mind putting them in a folder within the images directory instead of directly inside it?